static/src/javascripts/bower.json

```diff
@@ -36,7 +36,7 @@
       "lodash-amd": "bower_components/lodash-amd/compat/**",
       "domready": "bower_components/domready/ready.js",
       "when": "bower_components/when/es6-shim/Promise.js",
-      "video.js": "bower_components/video.js/dist/video.min.js",
+      "video.js": "bower_components/video.js/dist/video.js",
```

Comment:
You will also have to update the require config: https://github.com/guardian/frontend/blob/9e110256e0a2f07b6f39a78de42fb4273f9eb362/grunt-configs/requirejs.js#L26